### ISSUE TITLE
Service Unavailable

### ISSUE THREAD

**Issue by ricordisamoa** [toollabs/OctoData#3]
Malfunctions have been affecting the Deep Wiki Inspector. Should I just restart the OctoData webservice?

**Comment by Rillke** [comment]
Yes, just do whatever is required. I haven't contributed to the tool for a long time.

**Comment by ricordisamoa** [comment]
A new problem seems to be the s51816__krakenbase_p database missing required tables octoeye and octoeye_meta
